.travis.yml

```diff
@@ -9,7 +9,7 @@ before_install:
 - dep ensure
 
 install:
-- go get -t -d ./...
+- go get -t -d ./pkg/controllers/v1alpha1/...
 
 script:
-- go test ./...
+- go test ./pkg/controllers/v1alpha1/...
```

Comment:
Probably do something like
`go test $(go list ./... | grep -v /v1alpha2/)`

Reply:
Just updated to use `go test ./pkg/controllers/v1alpha1/... ./cmd/...` instead, excluding `v1alpha2` test execution for now.

Reply:
Actually I just realized there isn’t any test in cmd/, I just changed it back to “go test ./pkg/controllers/v1alpha1/...”? Are there any other tests that need to be run I may have missed?

Reply:
My point is we should explicitly exclude the v1alpha2 tests for now, so we don't forget to run other tests if/when we add them in the future.

Reply:
Got it. Done.